metadata.json

```diff
@@ -33,7 +33,7 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 4.10.0 < 7.0.0"
+      "version_requirement": ">= 5.5.8 < 7.0.0"
```

Comment:
5.5.10 is the latest in the 5 series

Reply:
https://voxpupuli.org/blog/2019/01/03/dropping-puppet4/ we decided on 5.5.8 back in the days. I'm not sure if we should increase the patch version or not.